src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -178,6 +184,7 @@ void assertPositiveAnonymizeCourtCaseEntity() {
         doNothing().when(dataAnonymisationService).anonymizeProsecutorEntity(any(), any());
         doNothing().when(dataAnonymisationService).anonymizeHearingEntity(any(), any());
         doNothing().when(dataAnonymisationService).tidyUpTransformedMediaEntities(any(), any());
+        doNothing().when(casesLoggerService).caseDeletedDueToExpiry(anyInt(), anyString());
```

Comment:
aren't these redundant? The default behaviour would be to do nothing

Reply:
removed all these unnecessary ones.